ydb/src/lib.rs

```diff
@@ -82,6 +82,9 @@ pub(crate) mod transaction;
 mod types;
 mod types_converters;
 pub(crate) mod waiter;
+mod client_topic;
```

Comment:
Add new public types to as "pub use ..." below please: By design of the crate all public-visibility types should be on top level and pub explicitly.